Tests/Integration/MobileCoinClientPublicAsyncApiIntTests.swift

```diff
@@ -14,24 +14,6 @@ import XCTest
 @available(iOS 15.0, *)
 class MobileCoinClientPublicAsyncApiIntTests: XCTestCase {
 
-    func testBalance() async throws {
```

Comment:
would prefer to keep this test and the balances test. the Print balances test is for debugging and will never fail

Reply:
we can `XCTSkip(true)` the printBalances test, and then we can use it when we need it by commenting that line in debuggin

Reply:
Since we have it available in async form for debugging, any reason why we need the non-async version???

We'll get the same info...

Reply:
Sure, but just not sure why we need it... if it's not testing anything, we can just use the async version.  Any reason to keep and maintain two versions of the functionality to just print balances?

Reply:
Oh, man... sorry - I just realized this is from the Async call, and you were referring to the removal of the single 'testBalance' async test.

Reply:
will put this one back and skip...

Reply:
added `testBalance` and `testPrintBalances` back, have them both set to XCTSkip() to save integration test execution time (balances are logged when updated for the tests as part of the update balance logic in any case)